src/py/flwr/common/serde.py

```diff
@@ -761,6 +762,7 @@ def metadata_from_proto(metadata_proto: ProtoMetadata) -> Metadata:
         ttl=metadata_proto.ttl,
         message_type=metadata_proto.message_type,
     )
+    metadata.created_at = metadata_proto.created_at
```

Comment:
This is here just because we have a test `metadata_from_proto` test. I think in a complete setup this will always be called from `message_from_proto` and therefore adding this last line to the function won't be needed